proposals/description/tip_121c_pt2.ts

```diff
@@ -0,0 +1,60 @@
+import { TemplatedProposalDescription } from '@custom-types/types';
+import { ethers } from 'ethers';
+
+// Asset amounts configuration
+const MAX_BASIS_POINTS = '10000'; // 100% in basis points
+const DAI_HOLDING_DEPOSIT_BALANCE = ethers.constants.WeiPerEther.mul(30_600_000); // TODO - update
```

Comment:
TO CHECK : what's the final amount of DAI left in the protocol, that can be sent to the TribeRedeemer ?